pkg/webhook/router.go

```diff
@@ -74,13 +74,16 @@ func (r *Router) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 func (r *Router) admit(response *Response, request *v1.AdmissionRequest, req *http.Request) error {
 	for _, m := range r.matches {
 		if m.matches(request) {
-			return m.admit(response, &Request{
+			err := m.admit(response, &Request{
 				AdmissionRequest: *request,
 				Context:          req.Context(),
 				objTemplate:      m.getObjType(),
 			})
+			logrus.Debugf("%s %s %s user=%s allowed=%v err=%v", request.Operation, request.Kind.String(), resourceString(request.Namespace, request.Name), request.UserInfo.Username, response.Allowed, err)
```

Comment:
Prefix the log message so we can see where it's coming from and have some context